config/initializers/carrier_wave.rb

```diff
@@ -0,0 +1,14 @@
+CarrierWave.configure do |config|
+  if Rails.env.staging? || Rails.env.production?
+    config.storage    = :aws
+    config.fog_credentials = {
+      provider:              'AWS',
+      aws_access_key_id:     ENV['AWS_ACCESS_KEY_ID'],
+      aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+    }
+    config.fog_directory  = 'myflixapp'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.